### PR TITLE
feat: no msw in production builds

### PR DESCRIPTION
**Pull request by chxash**
## Description

Fix for https://github.com/shapeshift/web/issues/840

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [X] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [X] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

 https://github.com/shapeshift/web/issues/840

## Risk

There should be no impact.

## Testing

For testing this, I verified that MSW was still effective even after removing the .js file in public directory.
To do that, I commented network mocks (jest.mock) in setupGlobalMocks.ts and verified that MSW was still capturing any network call during unit tests.

### CODE REVIEW

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
thanks for this - just requesting we don't modify the `yarn.lock` file in this PR as it seems unrelated?

File: yarn.lock [hunk 1]
```diff
@@ -5199,7 +5199,7 @@
   dependencies:
     "@types/unist" "*"
 
-"@types/history@*", "@types/history@^4.7.11":
+"@types/history@^4.7.11":
```
[the remark this answers is not in the document]
Comment (theoboldfrazier) [reply yarn.lock:5202]: This change looks to be in develop already. The github diff is off.

[the remark this answers is not in the document]
Comment (0xdef1cafe) [reply yarn.lock:5202]: ok cool, in that case @chxash can you please rebase off develop?

**Comment by chxash** [comment]
> thanks for this - just requesting we don't modify the `yarn.lock` file in this PR as it seems unrelated?

Done ! Sorry for this

File: yarn.lock (hunk 1 above)
[the remark this answers is not in the document]
Comment (chxash) [reply yarn.lock]: done !

**Review by 0xdef1cafe — APPROVED** [review]

**Review by gomesalexandre — APPROVED** [review]

**Pull request merged**